### PR TITLE
docs: Modify the introduction of the homepage Readme

### PR DESCRIPTION
**Pull request by zhenlei520**
### Docs
* Homepage ReadMe: Modify the introduction of the homepage Readme
* CQRS: Modify the introduction of the Readme
* Isolation.MultiEnvironment: Modify the introduction of the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,135 +6,77 @@
...
-> 4. Transaction
 
-> Effect：
->
-> 1. Event and Handler decoupling
-> 2. Arrangement of Handler
-> 3. Implement [Saga](https://docs.microsoft.com/zh-cn/azure/architecture/reference-architectures/saga/saga)
-> 4. Middleware
-> 5. Transaction
+## How to download
```
Comment (doddgu) [inline README.md:44]: I think this not download

File: README.zh-CN.md [hunk 2]
```diff
@@ -4,137 +4,79 @@
...
+  * [EFCore](./src/Data/Masa.Contrib.Data.UoW.EF/README.zh-CN.md)
+* Storage: 云存储
+  * [阿里云存储](./src/Storage/Masa.Contrib.Storage.ObjectStorage.Aliyun/README.zh-CN.md)
+* 业务能力
+  * 项目管理
+    * PM
+
+
+
+## 如何下载
```
Comment (doddgu) [inline README.zh-CN.md:44]: I think this not 下载

**Review by doddgu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (zhenlei520) [reply README.md:44]: How to Clone?

File: README.zh-CN.md (hunk 2 above)
Comment (zhenlei520) [reply README.zh-CN.md:44]: 如何克隆？

Comment (doddgu) [reply README.zh-CN.md]: How to build or develop?

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**